tests/upgrades/test_remoteexecution.py

```diff
@@ -0,0 +1,147 @@
+"""Test for Remote Execution related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import os
+
+from nailgun import entities
+from robottelo.constants import DEFAULT_LOC_ID, DISTRO_RHEL7
+from robottelo.helpers import add_remote_execution_ssh_key
+from robottelo.vm import VirtualMachine
+from robottelo.test import APITestCase
+from upgrade_tests import post_upgrade, pre_upgrade
+from upgrade_tests.helpers.scenarios import create_dict, get_entity_data
+
+
+class Scenario_remoteexecution_external_capsule(APITestCase):
+    """Test Remote Execution job created before migration runs successfully
+    post migration on a client registered with external capsule.
+
+        Test Steps:
+
+        1. Before Satellite upgrade:
+           a. Create Content host.
```

Comment:
We do only follow numbers but I hope this implementation wont affect makedocs. Avoid from next time.

Reply:
okay @jyejare